Provide basic README (#4)
SeanFeldman/azure-events-wishlist: README.md

# Azure Event Wishlist

> Azure emits a variety of events to Azure Event Grid which allows you to extend and react to what's going on in your apps & infrastructure, but not everything is there yet.

Reviewer: ```suggestion
Azure services emit a variety of events to Azure Event Grid enabling you to extend and react to what's going on in your applications and infrastructure. Although, not all the events are there yet.
```

> **Azure Event Wishlist** is a dashboard that gathers all events that would be great to have in Azure!

Reviewer: ```suggestion
**Azure Event Wishlist** is a single source curated dashboard that gathers all events that would be great to have in Azure! A single place to suggest, track, discuss, and get notified about new events added to various Azure services.
```

Author: Added!

## How can you help?

- **Propose new event suggestions** by opening a new issue

> - **Vote for an event** that you'd like to be supported by giving a :+1: on the issue

Reviewer: ```suggestion
- **Vote for the events** you'd like to see supported by giving a :+1: on the issues
```

Reviewer: Should the list have a third item, “**Discuss events** to provide valuable information to the Microsoft service teams”?

## Examples

> Here are examples of how powerful Azure events can be:

Reviewer: Plural, would need to be at least one more.

Author: Done
